packages/frontend-core/src/components/grid/controls/SizeButton.svelte

```diff
@@ -86,6 +87,7 @@
       <div class="options">
         {#each rowSizeOptions as option}
           <ActionButton
+            disabled={$fixedRowHeight}
```

Comment:
NAB: Is this needed? The Grid block doesn't have this button, unless I'm mistaken!